doc/api/buffer.md

```diff
@@ -1819,8 +1819,8 @@ console.log(buf.readUInt32LE(1).toString(16));
 added: v0.11.15
 -->
 
-* `offset` {integer} Where to start reading. Must satisfy: `0 <= offset <= buf.length - byteLength`
-* `byteLength` {integer} How many bytes to read. Must satisfy: `0 < byteLength <= 6`
+* `offset` {integer} Where to start reading. Must satisfy: `0 <= offset <= buf.length - byteLength`.
```

Comment:
I did 
`Where to start reading` -> `Number of bytes in at which to start reading.` and 
`How many bytes` -> `Number of bytes`

Let me know if you'd prefer something else.